stetho/src/main/java/com/facebook/stetho/common/android/ViewUtil.java

```diff
@@ -14,6 +19,85 @@ private ViewUtil() {
   }
 
   @Nullable
+  public static View hitTest(View view, float x, float y) {
+    return hitTest(view, x, y, null);
+  }
+
+  // x,y are in view's local coordinate space
+  @Nullable
+  public static View hitTest(View view, float x, float y, @Nullable Predicate<View> viewSelector) {
+    if (!ViewUtil.pointInView(view, x, y)) {
+      return null;
+    }
+
+    if (!(view instanceof ViewGroup)) {
+      return null;
+    }
+
+    final ViewGroup viewGroup = (ViewGroup)view;
+
+    // TODO: get list of Views that are sorted in Z- and draw-order, e.g. buildOrderedChildList()
+    for (int i = viewGroup.getChildCount() - 1; i >= 0; --i) {
+      final View child = viewGroup.getChildAt(i);
+
+      if (child.getVisibility() == View.INVISIBLE) {
+        continue;
+      }
+
+      if (viewSelector != null && !viewSelector.apply(child)) {
+        continue;
+      }
+
+      PointF localPoint = new PointF();
+      if (ViewUtil.isTransformedPointInView(viewGroup, child, x, y, localPoint)) {
+        if (child instanceof ViewGroup) {
+          return hitTest(child, localPoint.x, localPoint.y, viewSelector);
+        } else {
+          return child;
+        }
+      }
+    }
+
+    return viewGroup;
+  }
+
+  // see: View.pointInView()
+  public static boolean pointInView(View view, float localX, float localY) {
+    return localX >= 0 && localX < (view.getRight() - view.getLeft())
+        && localY >= 0 && localY < (view.getBottom() - view.getTop());
+  }
+
+  // see: ViewGroup.isTransformedTouchPointInView()
```

Comment:
hmmmmm unfortunately View.pointInView() is package-private (which is why I copied it here) and this doesn't seem to work ...